scripts/release/publish-docs-content.sh

```diff
@@ -5,10 +5,10 @@
 
 cd "$(dirname $0)/../../"
 
-docsPath="/dist/docs"
-repoPath="/tmp/material2-docs-content"
+docsPath="./dist/docs"
+repoPath="./tmp/material2-docs-content"
```

Comment:
This should be still /tmp. Just be sure to don't remove the whole `/tmp` dir later.